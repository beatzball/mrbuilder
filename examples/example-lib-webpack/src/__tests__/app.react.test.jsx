import React from 'react';
import App, {Smile} from '../index';
import renderer from 'react-test-renderer';

test('Should render lib', () => {
    const app = renderer.create(<App/>);
    expect(app.toJSON()).toMatchSnapshot();

});
test('Should render smile', () => {
    const smile = renderer.create(<Smile/>);
    expect(smile).toMatchSnapshot();
})