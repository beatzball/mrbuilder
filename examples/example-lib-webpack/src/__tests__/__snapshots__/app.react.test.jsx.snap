// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render lib 1`] = `
<div>
  <span
    className="container"
  >
    Hello
  </span>
   from webpack example.
</div>
`;
